tests/core/pyspec/eth2spec/test/altair/block_processing/sync_aggregate/test_process_sync_aggregate.py

```diff
@@ -201,6 +201,74 @@ def test_sync_committee_rewards_duplicate_committee_full_participation(spec, sta
     yield from run_successful_sync_committee_test(spec, state, committee_indices, committee_bits)
 
 
+@with_altair_and_later
+@with_presets([MAINNET], reason="to create duplicate committee")
+@spec_state_test
+def test_sync_committee_rewards_duplicate_committee_only_participate_first_one(spec, state):
```

Comment:
I know the name is already long, but I think we should include that it is 'zero_balance' somewhere in here as well.


Additionally, it might be good to have such a test case for a non-zero balance (although that's less interesting specifically here

Reply:
And in all cases, I suggest a quick helper defined right above to do the initial test setup.

or a helper test-runner that maybe takes `pre_balance, first_position[bool], second_position[bool]` as params. to deduplicate some of this code

Reply:
good ideas. addressed in c7e102a5e55ed32525e209d0a01ba5ee7d0ae82a.

Reply:
`good`